frontend/src/component/changeRequest/ChangeRequest/Changes/Change/Conflict.tsx

```diff
@@ -0,0 +1,24 @@
+import { Alert } from '@mui/material';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+
+export const ConflictWarning: React.FC<{
```

Comment:
Wonder whether this should be renamed into "ConditionalConflictWarning" or something similar? Input is much appreciated.